imap_processing/tests/test_cli.py

```diff
@@ -28,7 +29,14 @@ def mock_instrument_dependencies():
 
 @mock.patch("imap_processing.cli.Mag")
 def test_main(mock_instrument):
-    """Test imap_processing.cli.main()"""
+    """
+    Test imap_processing.cli.main().
+
+    Parameters
+    ----------
+    mock_instrument : str TODO CHeck
```

Comment:
```suggestion
    mock_instrument : str
```
Yes, I believe this is of `str` type